deliver/lib/deliver/loader.rb

```diff
@@ -9,10 +9,32 @@ module Loader
     DEFAULT_DIR_NAME = "default".freeze
     ALL_LANGUAGES = (FastlaneCore::Languages::ALL_LANGUAGES + [APPLE_TV_DIR_NAME, APPLE_TV_DIR_NAME, IMESSAGE_DIR_NAME, DEFAULT_DIR_NAME]).map(&:downcase).freeze
 
-    def self.language_folders(root)
-      Dir.glob(File.join(root, '*')).select do |path|
+    EXCEPTION_DIRECTORIES = UploadMetadata::ALL_META_SUB_DIRS.map(&:downcase).freeze
+
+    def self.language_folders(root, skip_unsupported_languages = true)
+      folders = Dir.glob(File.join(root, '*'))
+
+      selected_folders = folders.select do |path|
         File.directory?(path) && ALL_LANGUAGES.include?(File.basename(path).downcase)
```

Comment:
I wonder if we should fetch these from `Spaceship::Tunes::Client#available_languages` and cache them on the class? That way they will always be up to date with what Apple is expecting?